src/dispatch/plugins/dispatch_slack/commands.py

```diff
@@ -161,50 +168,68 @@ def filter_tasks_by_assignee_and_creator(tasks: List[Task], by_assignee: str, by
     return filtered_tasks
 
 
-async def handle_slack_command(*, client, request, background_tasks):
-    """Handles slack command message."""
-    # We fetch conversation by channel id
-    channel_id = request.get("channel_id")
-
-    db_session = get_organization_from_channel_id(channel_id=channel_id)
-    # We get the name of command that was run
+async def handle_non_incident_conversation_commands(client, request, background_tasks):
+    """Handles all commands that do not have a specific incident conversation."""
     command = request.get("command")
+    channel_id = request.get("channel_id")
+    command_args = request.get("text", "").split(" ")
+    if len(command_args):
+        organization_slug = command_args[0]
```

Comment:
Should we check if it's a valid organization slug and if not let the user know before assigning it?

Reply:
That validation happens centrally here: https://github.com/Netflix/dispatch/pull/1551/files#diff-7ccd0eb2d6907a9ee039fbd73281d9ea22bd3f7f8e5948a4beff61043ff7d16dR63